01week/rockPaperScissors.js

```diff
@@ -7,17 +7,40 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+const rockPaperScissors=(hand1, hand2)=> {
+  const aTie = "Its's a tie!";
+  const oneWins = "Hand one wins!";
+  const twoWins = "Hand two wins!";
 
-function rockPaperScissors(hand1, hand2) {
+  if (hand1 === hand2) {
+    return aTie;
+  } else if (hand1 === 'rock') {
+    if (hand2 === 'paper') {
+      return twoWins;
+    } else {
+      return oneWins;
+    }
 
-  // Write code here
+  } else if (hand1 === 'paper') {
+    if (hand2 === 'rock') {
+      return oneWins;
+    } else {
+      return twoWins;
+    }
 
+  }
+  // hand1 must be Scissors
+  if (hand2 === 'rock') {
```

Comment:
This is a short way of writing the logic but don't assume your users will use just rock, paper or scissors. Be sure to check for valid input and convert them to uniform strings.